src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState, useRef } from 'react';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoItem: React.FC<Props> = ({ todo }) => {
+  const { title, completed } = todo;
+  const [isEditable, setIsEditable] = useState(false);
+  const [value, setValue] = useState(title);
+  const inputEditRef = useRef<HTMLInputElement | null>(null);
+
+  const handeleClickOnTitle = () => {
+    setIsEditable(true);
+
+    setTimeout(() => {
+      inputEditRef.current?.focus();
+    }, 0);
```

Comment:
Do you need `setTimeout`?

Reply:
yes, in example error hide after some time

Reply:
Yes, without setTimeOut focus not work, because inputRef.current not get on time